### PR TITLE
Trainee can see all Sessions

### PR DESCRIPTION
**Pull request by johannaanden**
# PT Link

[Trainee can see all Sessions](https://www.pivotaltracker.com/story/show/163530928)

# User Story

```
As a Trainee
In order to be able to choose the session I want
I would like to see all the available sessions on the home page
```

# Screenshots

jan-29-2019 15-09-21

# Description of PR

Adds status to all session (available, booked and full) and shows the to index page

### CODE REVIEW

File: spec/factories/sessions.rb [hunk 1]
```diff
@@ -0,0 +1,7 @@
+FactoryBot.define do
+  factory :session do
+			title { "CrossFit" }
```
Comment (HannaSwDn) [inline spec/factories/sessions.rb]: Indentation

File: spec/models/session_spec.rb [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+    it { is_expected.to validate_presence_of :title }
+    it { is_expected.to validate_presence_of :start_date }
+  end
+
+  describe FactoryBot do
+    it 'should have valid Factory' do
+      expect(create(:session)).to be_valid
+    end
+  end
+
```
Comment (HannaSwDn) [inline spec/models/session_spec.rb]: remove this line please

**Review by schneiderman805 — APPROVED** [review]
looks all good to me

File: spec/factories/sessions.rb (hunk 1 above)
Comment (johannaanden) [reply spec/factories/sessions.rb]: Done

File: spec/models/session_spec.rb (hunk 2 above)
Comment (johannaanden) [reply spec/models/session_spec.rb]: Done

**Review by HannaSwDn — APPROVED** [review]

**Pull request merged**